src/pages/benchviz/index.tsx

```diff
@@ -0,0 +1,16 @@
+import Layout from "@theme/Layout";
+import React from "react";
+
+const Benchmark = React.lazy(() => import("./Benchmark"));
```

Comment:
In case of /play it was done because monaco couldn't even be imported out of browser. Unless `d3` works just like that, it could just be a simple component rendering the page body